README.md

```diff
@@ -1,2 +1,3 @@
 FLAVORTRIPPINGBABY
 ===
+https://twitter.com/pakalupapitow/status/1167168200003317761
```

Comment:
@barcher want me to take it down?

Reply:
@bsiddiqui Nah, all good. Just 👍 my PR 😛